conftest.py

```diff
@@ -344,7 +380,20 @@ def bank_account(pytestconfig: Config) -> Generator[Keypair | None, None, None]:
             raise ValueError("set BANK_PRIVATE_KEY or BANK_PRIVATE_KEY_MAINNET env variable")
         key = base58.b58decode(private_key)
         account = Keypair.from_bytes(key)
+
+    if pytestconfig.environment.use_bank:
+        ata = sol_client_session.create_associate_token_acc(account, account, WRAPPED_SOL_MINT)
+
     yield account
+    if pytestconfig.environment.use_bank:
+        balance = sol_client_session.get_solana_balance(account.pubkey())
+        spl_token = SplToken(sol_client_session, WRAPPED_SOL_MINT, TOKEN_PROGRAM_ID, account)
+        ata_balance = int(spl_token.get_balance(ata, commitment=Confirmed).value.amount)
+        spl_token.close_account(account=ata, dest=account.pubkey(), authority=account)
+        wait_condition(
```

Comment:
I don't think wee need checks like that here (as close_account is solana functionality)
The check can fail as we use the same bank account for all threads in tests run and balance is changing in parallel trx

Reply:
removed